### DISCUSSION TITLE
Expose call to get saved layouts

### DISCUSSION THREAD

**Discussion by rtriunfo** [rxhanson/RectanglePro-Community#461] [Ideas]
Provide a way to retrieve saved layouts from rectangle pro. Possibly: 
rectangle://getLayouts

**Comment by rxhanson** [comment]
Thanks for the feedback. I'll add this to my backlog, but it's unlikely I will implement this unless there is significant demand. In the meantime if you're handy with scripting, you could parse the plist where the defaults are stored and obtain it that way: 
```
~/Library/Preferences/com.knollsoft.Hookshot.plist
```
